Library/Homebrew/dependable.rb

```diff
@@ -11,6 +11,8 @@ module Dependable
   # misuse in future.
   RESERVED_TAGS = [:build, :optional, :recommended, :run, :test, :linked].freeze
 
+  attr_reader :tags
```

Comment:
This is a bit tangential, but moving the `attr` here reduces duplication, allows us to remove the matching `.rbi` file, and write better tests.